bees/execbee/execbee.go

```diff
@@ -45,13 +47,31 @@ func (mod *ExecBee) Action(action bees.Action) []bees.Placeholder {
 	switch action.Name {
 	case "execute":
 		var command string
+		var stdin string
+
 		action.Options.Bind("command", &command)
+		action.Options.Bind("stdin", &stdin)
 		mod.Logln("Executing locally: ", command)
 
 		go func() {
 			c := strings.Split(command, " ")
 			cmd := exec.Command(c[0], c[1:]...)
 
+			if stdin != "" {
+				stdinPipe, err := cmd.StdinPipe()
+
+				if err != nil {
+					mod.LogFatal("Error creating stdinPipe for Cmd", err)
+					return
+				}
+
+				go func() {
+					mod.Logf("StdIn" + stdin)
```

Comment:
The supplied options / parameters are usually already printed out. I guess we could remove it here.